### PR TITLE
Update astar.py

### PR DESCRIPTION
**Pull request by ahtesham007**
Improved comments added punctuations.

### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: machine_learning/astar.py [hunk 1]
```diff
@@ -137,14 +137,14 @@ def astar(world, start, goal):
 
 if __name__ == "__main__":
     world = Gridworld()
-    #   stat position and Goal
+    #   Stat position and goal.
```
Comment (CaedenPH) [inline machine_learning/astar.py:140]:
```suggestion
    #   Stat position and goal
```
You dont really need punctuation in a comment

File: machine_learning/astar.py [hunk 2]
```diff
@@ -137,14 +137,14 @@ def astar(world, start, goal):
...
     start = Cell()
     start.position = (0, 0)
     goal = Cell()
     goal.position = (4, 4)
     print(f"path from {start.position} to {goal.position}")
     s = astar(world, start, goal)
-    #   Just for visual reasons
+    #   Just for visual reasons.
```
Comment (CaedenPH) [inline machine_learning/astar.py:147]:
```suggestion
    #   Just for visual reasons
```

File: machine_learning/astar.py (hunk 1 above)
Comment (ahtesham007) [reply machine_learning/astar.py:140]: Got it. But the comments were inconsistent. Lower & Upper case words, either it should all be lower cased or proper english sentences.

**Review by cclauss — APPROVED** [review]
LGTM

**Pull request merged**